### DISCUSSION TITLE
使用grobid预处理PDF页面，改善PDF翻译

### DISCUSSION THREAD

**Discussion by Geo123abc** [immersive-translate/immersive-translate#743] [Ideas]
### 新功能 | New Feature

[zotero style 2.7.0 ](  https://github.com/MuiseDestiny/zotero-style/releases/tag/2.7.0  )    实现了PDF全文翻译（视频：[为了吸引你，我为 Zotero 写了全文翻译功能 - 不一样的全文翻译，功能永久免费_哔哩哔哩_bilibili](https://www.bilibili.com/video/BV14h4y127TB/)），其对PDF页面处理采用了 [kermitt2/grobid: A machine learning software for extracting information from scholarly documents](  https://github.com/kermitt2/grobid  )    ，可能会对沉浸式翻译改善识别PDF界面有所帮助（或者沉浸式翻译已经使用了这个tool？）

### 为什么您想要添加这些功能 | Why you want to add these features

PDF页面按照段落预处理之后，应该可以极大减少GPT的请求次数，改善GPT的翻译体验

**Comment by theowenyoung** [comment]
zetero style的源码似乎比 release的版本旧？这个插件目前是闭源了吗？

**Comment by Geo123abc** [comment]
> zetero style的源码似乎比 release的版本旧？这个插件目前是闭源了吗？

抱歉，我没注意到这个，您可以问一下开发者，我是最近用到了这个，感觉产品之间可以相互借鉴一下
